### PR TITLE
change the wrong file name and command 

### PR DESCRIPTION
**Pull request by kiwoongko**
change the wrong file name to real name in the toolchain server and change command to unzip bz2 and ubuntu x64.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,21 +6,34 @@
...
+
+```bash
+sudo apt-get -y install lib32ncurses5
+```
+
 ### Getting the sources
 
 ```bash
-git clone https://github.com/Samsung/TizenRT.git
+git clone https://github.sec.samsung.net.com/RS7-OpensourceGroup/tizenrt.git
```
Comment (sunghan-chang) [inline README.md]: What is this?

File: README.md [hunk 2]
```diff
@@ -6,21 +6,34 @@
 lightweight RTOS-based platform to support low-end IoT devices.  
 Please find project details like **APIs**, **Specification** and **Long-term Goals** on our [Tizen Site](https://source.tizen.org/documentation/tizen-rt).
 
+## Contents
```
Comment (sunghan-chang) [inline README.md]: This was already merged. I think you use an old version.

File: README.md [hunk 3]
```diff
@@ -71,9 +84,9 @@ Please keep in mind that we are actively working on board configurations, and wi
 ## APPENDIX
 ### Kconfig-frontends Installation
 
-1. The *gperf* and *libncurses5-dev* packages should be installed.
+1. The *byacc*, *flex*, *gperf* and *libncurses5-dev* packages should be installed.
```
Comment (sunghan-chang) [inline README.md]: same as previous

File: README.md [hunk 4]
```diff
@@ -6,21 +6,34 @@
...
+Get the build in binaries and libraries, [gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2](https://launchpad.net/gcc-arm-embedded/4.9/4.9-2015-q3-update)  
+Unzip and untar the gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2 and export the path like
 
 ```bash
-tar xvf gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar
+tar xvjf gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2
 export PATH=<Your Toolchain PATH>:$PATH
 ```
 
+if you run on ubuntu x64,
```
Comment (sunghan-chang) [inline README.md]: Is this needed for toolchain?

**Comment by kiwoongko** [comment]
my mistake. I requested with old version.

**Pull request closed without merging**